Makefile

```diff
@@ -201,7 +208,7 @@ push-image-gce:
 
 .PHONY: image-test
 image-test:
-	@docker run --rm -v /dev:/dev -v /tmp:/out --privileged $(DOCKER_ARGS) autonomy/installer:$(TAG) install -n test -r -p test -u none
```

Comment:
Dropping the output dir mount because we don't really care about the image afterwards.

Reply:
Let’s keep the output dir. I can still see the value in being able to run something like fdisk on the test image. Plus it more accurately tests the way we generate images.